src/main/java/org/reaktivity/nukleus/socks/internal/stream/Correlation.java

```diff
@@ -0,0 +1,241 @@
+    {
+        this.acceptCorrelationId = correlationId;
+    }
+
+    public String acceptSourceName()
+    {
+        return acceptSourceName;
+    }
+
+    public void acceptSourceName(String acceptSourceName)
+    {
+        this.acceptSourceName = acceptSourceName;
+    }
+
+    public long connectStreamId()
+    {
+        return connectStreamId;
+    }
+
+    public void connectStreamId(long connectStreamId)
+    {
+        this.connectStreamId = connectStreamId;
+    }
+
+    public long connectTargetRef()
+    {
+        return connectTargetRef;
+    }
+
+    public void connectTargetRef(long connectTargetRef)
+    {
+        this.connectTargetRef = connectTargetRef;
+    }
+
+    public MessageConsumer connectEndpoint()
+    {
+        return connectEndpoint;
+    }
+
+    public void connectEndpoint(MessageConsumer connectEndpoint)
+    {
+        this.connectEndpoint = connectEndpoint;
+    }
+
+    public MessageConsumer acceptThrottle()
+    {
+        return acceptThrottle;
+    }
+
+    public void acceptThrottle(MessageConsumer acceptThrottle)
+    {
+        this.acceptThrottle = acceptThrottle;
+    }
+
+    public long acceptStreamId()
+    {
+        return acceptStreamId;
+    }
+
+    public void acceptStreamId(long acceptStreamId)
+    {
+        this.acceptStreamId = acceptStreamId;
+    }
+
+    public long acceptSourceRef()
+    {
+        return acceptSourceRef;
+    }
+
+    public void acceptSourceRef(long acceptSourceRef)
+    {
+        this.acceptSourceRef = acceptSourceRef;
+    }
+
+    public MessageConsumer acceptReplyEndpoint()
+    {
+        return acceptReplyEndpoint;
+    }
+
+    public void acceptReplyEndpoint(MessageConsumer acceptReplyEndpoint)
+    {
+        this.acceptReplyEndpoint = acceptReplyEndpoint;
+    }
+
+    public long acceptReplyStreamId()
+    {
+        return acceptReplyStreamId;
+    }
+
+    public void acceptReplyStreamId(long acceptReplyStreamId)
+    {
+        this.acceptReplyStreamId = acceptReplyStreamId;
+    }
+
+    public String connectTargetName()
+    {
+        return connectTargetName;
+    }
+
+    public void connectTargetName(String connectTargetName)
+    {
+        this.connectTargetName = connectTargetName;
+    }
+
+    public long connectCorrelationId()
+    {
+        return connectCorrelationId;
+    }
+
+    public void connectCorrelationId(long connectCorrelationId)
+    {
+        this.connectCorrelationId = connectCorrelationId;
+    }
+
+    public RouteFW connectRoute()
+    {
+        return connectRoute;
+    }
+
+    public void connectRoute(RouteFW connectRoute)
+    {
+        this.connectRoute = connectRoute;
+    }
+
+    public Consumer<Boolean> nextAcceptSignal()
+    {
+        return nextAcceptSignal;
+    }
+
+    public void nextAcceptSignal(Consumer<Boolean> nextAcceptSignal)
+    {
+        this.nextAcceptSignal = nextAcceptSignal;
+    }
+
+    public MessageConsumer connectReplyThrottle()
```

Comment:
I'd be surprised if the connectReply were used for passing back and forth on the stream types...?